src/odemis/driver/ueye.py

```diff
@@ -646,14 +679,20 @@ def __getitem__(self, name):
 
 class Camera(model.DigitalCamera):
     """
-    Represents a IDS uEye camera.
+    Represents an IDS uEye camera.
     Currently, only greyscale mode is supported
     """
 
-    def __init__(self, name, role, device=None, **kwargs):
+    def __init__(self, name, role, device=None, max_res=None, **kwargs):
         """
         device (None or str): serial number (eg, 1020345) of the device to use
-          or None if any device is fine.
+        or None if any device is fine.
```

Comment:
Typically, in doctrings, we use two spaces indentation to indicate that it continues the line before.

Reply:
The PEP 8 standard for docstrings refers to PEP 257 and in there it describes multiline docstrings with an example:
def bar():
    """
    A multi-line
    docstring.
    """
So I am a little confused. In my mind I wanted to start that line just below the first part with the same indentation.
If you say we always have to use 2 spaces in this case, we might also add that to Karishma's PR for coding guidelines.

Reply:
I think that is more "my" style than the official one, indeed. I used to directly the parameter name, without `:param `, hence it was handy to have some indentation to separate the parameters.